content/vanus-ai/how-to/discord.md

```diff
@@ -0,0 +1,54 @@
+# Discord Integration
+This documentation will teach you how to integrate an AI into Discord.
+Integration is a paid user feature. Therefore, only users with subscriptions or 14 days trials can use this feature.
+
+## how to Setup a Discord Integration
+
+### Step 1: Create an AI Application
+1. Go to [ai.vanus.ai](https://ai.vanus.ai) and sign in.
+2. Create an new AI app by clicking **Create AI**.
+![](images/discord-createapp.png)
+3. Enter a name, the model, upload to the knowledge base, and click **Create**.
+![](images/discord-enterNameModelKnownledge.png)
+4. Go to the **integration** TAB and click on the discord integration.
+![](images/discord-integration.png)
+
+### Step 2: Create a Discord Channel
+1. Go to the discord App or website at [discord](https://discord.com)
+2. Once logged in you can click on the **`+`** button to create a new channel.
+![](images/discord-createChannel.png)
+3. Click **Create my own** and complete the creation of your channel.
+
+### Step 3: Create a Discord Application
+1. Visit [Discord Developer Portal](https://discord.com/developers/applications)
+2. Click **New Application**.
+![](images/discord-newapplication.png)
+3. Give a name to your bot, accept the terms and conditions and click **Create**.
+![](images/discord-givebotname.png)
+4. From the side menu go to **bot**, and in Privileged Gateway Intents enable **MESSAGE CONTENT INTENT**.
+![](images/discord-sidemenuclickbot.png)
+5. Click **Reset token** and **Copy**.
+![](images/discord-resettoken.png)
+6. Go back to Vanus AI paste the Token, give the integration a name and press Create.
```

Comment:
Go back to Vanus AI, paste the token, give the integration a name, and press Create.